dss/operations/checkout.py

```diff
@@ -24,35 +24,34 @@
 
 class CheckoutHandler:
     def __init__(self, argv: typing.List[str], args: argparse.Namespace):
-        if args.keys is not None:
-            self.keys = args.keys.copy()
-        else:
-            self.keys = None
+        self.keys = args.keys.copy()
         self.replica = Replica[args.replica]
         self.handle = Config.get_blobstore_handle(self.replica)
-        self.bucket = self.replica.checkout_bucket
+        self.checkout_bucket = self.replica.checkout_bucket
 
-    def _verify_delete(self, handle, bucket, key):
-        logger.warning(f'attempting removal of key: {bucket}/{key}')
+    @staticmethod
+    def _verify_delete(handle, bucket, key):
+        logger.warning(f'Attempting removal of key: {bucket}/{key}')
         try:
             handle.delete(bucket=bucket, key=key)
             handle.get(bucket=bucket, key=key)
         except BlobNotFoundError:
             logger.warning(f'Successfully deleted key {bucket}/{key}')
         else:
-            raise RuntimeError(f'attempted to delete {bucket}/{key} but it did not work ;(')
+            raise RuntimeError(f'Attempted to delete {bucket}/{key} but it did not work ;(')
 
-    def _verify_get(self, handle, bucket, key):
-        """ Helper Function to get file_metadata from cloud-blobstore"""
+    @staticmethod
+    def _get_metadata(handle, bucket, key):
+        """Helper Function to get file_metadata from cloud-blobstore."""
         try:
             file_metadata = json.loads(handle.get(bucket=bucket, key=key).decode('utf-8'))
+            return file_metadata
         except BlobNotFoundError:
-            logger.warning(f'unable to locate {bucket}/{key}')
-            return None
-        return file_metadata
+            logger.warning(f'Unable to locate: {bucket}/{key}')
```

Comment:
return `None` was removed here, does json.loads(handle.get()) eval to  None if there is no blob found ?

Reply:
Yes.  The default of any function that does not explicitly return is to return `None`.

Reply:
Does this need to get changed to sys.stderr.write() ?